### PR TITLE
[Outlook] (quick start) Miscellaneous updates to code snippets and instructions.

### PR DESCRIPTION
**Pull request by kbrandl**
Addresses issues #143 and #322.

### CODE REVIEW

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[57d206a](https://github.com/OfficeDev/outlook-dev-docs/pull/475/commits/57d206a89cd8cdaa31eb5aff4831b509acbf6d1f)_: 

:clock10: **Preparing**: average preparing time is _41 sec(s)_

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[57d206a](https://github.com/OfficeDev/outlook-dev-docs/pull/475/commits/57d206a89cd8cdaa31eb5aff4831b509acbf6d1f)_: 

:clock10: **Incremental building**: average incremental building time is _18 sec(s)_

**Comment by jasonjoh** [comment]
OPS Build status updates of commit _[57d206a](https://github.com/OfficeDev/outlook-dev-docs/pull/475/commits/57d206a89cd8cdaa31eb5aff4831b509acbf6d1f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/quick-start.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-qs-updates-201812/docs/add-ins/quick-start.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C27%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201812272347512893-475%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=lFEURK7WMeBUQQgcAZA02GRzXZVK7P62p%2BqhG0sOMxs%3D&st=2018-12-27T23%3A43%3A50Z&se=2019-01-27T23%3A48%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/add-ins/quick-start.md [hunk 1]
```diff
@@ -151,14 +157,12 @@ When you've completed the wizard, Visual Studio creates a solution that contains
...
+
+1. In the **Connect to Exchange email account** dialog box, enter the email address and password for your [Microsoft account](https://account.microsoft.com/account) and then choose **Connect**. When the Outlook.com login page opens in a browser, login to your email account with the same credentials as you entered previously.
 
     > [!NOTE]
-    > If you're asked to log in repeatedly then Basic Auth may be disabled for accounts on your Office 365 tenant. You can consider:
-    >
-    > a. hosting the add-in on a local server then [sideloading](sideload-outlook-add-ins-for-testing.md).
-    >
-    > b. using a Microsoft account instead.
+    > If the **Connect to Exchange email account** dialog box repeatedly prompts you to login, Basic Auth may be disabled for accounts on your Office 365 tenant. To test this add-in, login using a [Microsoft account](https://account.microsoft.com/account) instead.
```
Comment (AlexJerabek) [inline docs/add-ins/quick-start.md]: Is Basic Auth a term users of this quick start will be familiar with?

**Pull request merged**